main.go

```diff
@@ -133,6 +145,7 @@ func main() {
 		}
 
 		fmt.Fprintf(stdout, "Summary:\n")
+		fmt.Fprintf(stdout, "Zone checksum: %s\n", zoneChecksum)
```

Comment:
How about "SHA256 zone checksum" to make it explicit what the value represent?

Reply:
Added